src/momento/typing.py

```diff
@@ -10,6 +10,14 @@
 TCollectionName = str
 
 # Dictionary Types
+TDictionaryName = TCollectionName
+TDictionaryField = Union[str, bytes]
+TDictionaryValue = Union[str, bytes]
+TDictionaryFields = Sequence[TDictionaryField]
```

Comment:
We're going with Iterator for listy inputs, correct?

```suggestion
TDictionaryFields = Iterator[TDictionaryField]
```

Reply:
Yes thanks

Reply:
Fixed in [65ab746](https://github.com/momentohq/client-sdk-python/pull/229/commits/65ab746868843ba16c8b99aeb76b1d3c5728fa0e)